Tests/configure_and_test_integration_instances.py

```diff
@@ -959,70 +1122,134 @@ def main():
+    #     client = demisto_client.configure(base_url=server_url, username=build.username, password=build.password, verify_ssl=False)
+    #     __disable_integrations_instances(client, all_module_instances, prints_manager)
+    client = demisto_client.configure(base_url=build.servers[0], username=build.username, password=build.password,
+                                      verify_ssl=False)
+    __disable_integrations_instances(client, all_module_instances, prints_manager)
     prints_manager.execute_thread_prints(0)
 
-    for instance in all_module_instances:
-        integration_of_instance = instance.get('brand', '')
-        instance_name = instance.get('name', '')
-        msg = 'Testing ("Test" button) for instance "{}" of integration "{}" .'.format(instance_name,
-                                                                                       integration_of_instance)
-        prints_manager.add_print_job(msg, print_color, 0, LOG_COLORS.GREEN)
-        prints_manager.execute_thread_prints(0)
-        # If there is a failure, __test_integration_instance will print it
-        success, _ = __test_integration_instance(testing_client, instance, prints_manager)
-        prints_manager.execute_thread_prints(0)
-        if not success:
-            postupdate_fails.add((instance_name, integration_of_instance))
+
+def create_nightly_test_pack():
+    test_pack_zip(Build.content_path, Build.test_pack_target)
+
+
+def test_files(content_path):
+    packs_root, packs, _ = next(os.walk(f'{content_path}/Packs')) # lgtm [py/unguarded-next-in-generator]
+    for pack_dir in packs:
+        if pack_dir in SKIPPED_PACKS:
+            continue
+        if os.path.isdir(f'{packs_root}/{pack_dir}/TestPlaybooks'):
+            playbooks_root, _, playbooks = next(os.walk(f'{packs_root}/{pack_dir}/TestPlaybooks')) # lgtm [py/unguarded-next-in-generator]
+            for playbook in playbooks:
+                yield os.path.join(playbooks_root, playbook), playbook
+
+
+def test_pack_metadata():
+    now = datetime.now().isoformat().split('.')[0]
+    now = f'{now}Z'
+    metadata = {
+        "name": "nightly test",
+        "id": str(uuid.uuid4()),
+        "description": "nightly test pack (all test playbooks and scripts).",
+        "created": now,
+        "updated": now,
+        "legacy": True,
+        "support": "Cortex XSOAR",
+        "supportDetails": {},
+        "author": "Cortex XSOAR",
+        "authorImage": "",
+        "certification": "certified",
+        "price": 0,
+        "serverMinVersion": "6.0.0",
+        "serverLicense": "",
+        "currentVersion": "1.0.0",
+        "general": [],
+        "tags": [],
+        "categories": [
+            "Forensics & Malware Analysis"
+        ],
+        "contentItems": {},
+        "integrations": [],
+        "useCases": [],
+        "keywords": [],
+        "dependencies": {}
+    }
+    return json.dumps(metadata, indent=4)
+
+
+def test_pack_zip(content_path, target):
```

Comment:
Are we leaving this part here or it will be moved to create content artifacts?

Reply:
for now, it’s here but it’s in the plan to move.